sdk/src/client/node_api/core/routes.rs

```diff
@@ -294,15 +294,15 @@ impl ClientInner {
             .await
     }
 
-    /// Get the metadata for a given `OutputId` (TransactionId + output_index).
+    /// Finds output metadata by its identifier.
```

Comment:
total NIP: `its identifier` suggests  `output metdata's` identifier. Maybe ` by output identifier` or something similar?

Reply:
I can change it, even though there is no such thing as metadata identifier so it shouldn't be confusing

Reply:
done